selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -617,16 +656,16 @@ public void waitTextIntoAutocompleteContainer(final String value) {
    * @param position line`s number where marker is expected
    */
   public void waitMarkerInPosition(MarkerLocator markerLocator, int position) {
-    elemDriverWait.until(
-        visibilityOfElementLocated(By.xpath(format(markerLocator.get(), position))));
+    seleniumWebDriverHelper.waitVisibility(
+        By.xpath(format(markerLocator.get(), position)), ELEMENT_TIMEOUT_SEC);
     setCursorToLine(position);
     expectedNumberOfActiveLine(position);
   }
 
   /** Wait for no Git change markers in the opened editor. */
   public void waitNoGitChangeMarkers() {
-
-    new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC)
+    webDriverWaitFactory
+        .get(DEFAULT_TIMEOUT)
```

Comment:
We can omit _DEFAULT_TIMEOUT_ parameter by creation `WebDriverWaitFactory#get()` method which uses `DEFAULT_TIMEOUT` inside.

Reply:
Good point, thanks for the idea.